src/schema/feeds.ts

```diff
@@ -1203,6 +1205,8 @@ const feedResolverCursor = feedResolver<
         ...(args?.refresh && { refresh: true }),
       }),
     warnOnPartialFirstPage: true,
+    // Feed service should take care of this
+    removeNonPublicThresholdSquads: false,
```

Comment:
But still why do we ship it here then?

Reply:
I disabled this condition for the feed to make the query leaner on the API side. It is enabled by default